test/e2e_node/runtime_conformance_test.go

```diff
@@ -18,233 +18,23 @@ package e2e_node
 
 import (
 	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"io/ioutil"
-	"os"
-	"path"
-	"path/filepath"
-	"time"
-
 	"k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/kubernetes/pkg/kubelet/images"
+	"k8s.io/kubernetes/test/e2e/common"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e_node/services"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-	gomegatypes "github.com/onsi/gomega/types"
-)
-
-const (
-	consistentCheckTimeout = time.Second * 5
-	retryTimeout           = time.Minute * 5
-	pollInterval           = time.Second * 1
+	"os"
+	"path/filepath"
+	"time"
```

Comment:
Style nit: taking a look at other files, the style seems to be:
```
import (
  [built-ins]
  <space>
  [k8s.io/... ones]
  <space>
  [other non built-ins like github ones]
)
```

Reply:
Done.